### PR TITLE
libudev: coding style updates

### PR DESCRIPTION
**Pull request by yuwata**

### CODE REVIEW

File: src/libudev/libudev.c [hunk 1]
```diff
@@ -109,16 +103,9 @@ _public_ struct udev *udev_ref(struct udev *udev) {
...
- * Returns: the passed udev library context if it has still an active reference, or #NULL otherwise.
+ * Returns: #NULL.
  **/
-_public_ struct udev *udev_unref(struct udev *udev) {
-        if (udev == NULL)
-                return NULL;
-        udev->refcount--;
-        if (udev->refcount > 0)
-                return udev;
-        return mfree(udev);
```
Comment (keszybz) [inline src/libudev/libudev.c]: Nope, this is not backwards compatible.

(For example, someone could do cleanup in this way:
```C
while ((udev = udev_unref(udev)));
```
With this change, this becomes broken.)

Comment (yuwata) [reply src/libudev/libudev.c]: OK. Note that the man page udev_unref(3) states that 'udev_unref() always returns NULL.' Should we update this?

**Comment by yuwata** [comment]
Updated. The change in `udev_unref()` is dropped.

File: src/libudev/libudev-monitor.c [hunk 2]
```diff
@@ -233,26 +233,23 @@ _public_ struct udev_monitor *udev_monitor_new_from_netlink(struct udev *udev, c
...
-
-        ins->code = code;
-        ins->k = data;
-        (*i)++;
+static void bpf_stmt(struct sock_filter *ins, unsigned *i,
+                     unsigned short code, unsigned data) {
+        ins[(*i)++] = (struct sock_filter) {
+                .code = code,
+                .k = data,
+        };
```
Comment (keszybz) [inline src/libudev/libudev-monitor.c:205]: We might even want to simply get rid of those two helpers, I don't think they help much at this point.

**Pull request merged**